packages/ibm-products-web-components/README.md

```diff
@@ -1,5 +1,93 @@
 # @carbon/ibm-products-web-components
 
+> Carbon for IBM Products web components
+
+## Getting started
+
+If you’re just getting started and looking to browse our React components, take
+a look at
+[our Storybook](https://ibm-products-web-components.carbondesignsystem.com/).
+
+### 📦 Installing Carbon for IBM Products web components
+
+To use Carbon for IBM Products web components, all you need to do is install the
+`@carbon/ibm-products-web-components` package.
+
+```shell
+$ yarn add @carbon/ibm-products-web-components
+
+# or
+
+$ npm install @carbon/ibm-products-web-components
+```
+
+Then you can import the component styles in your `index.js`.
+
+```js
+import '@carbon/ibm-products/css/index.min.css';
+```
+
+### Usage
+
+The `@carbon/web-components` package provides components for the Carbon Design
```

Comment:
I would remove this line, or update the content to be specific to our package rather than `@carbon/web-components`.